ert_data/measured.py

```diff
@@ -1,154 +1,196 @@
+from collections import defaultdict
+
+import deprecation
 import pandas as pd
+import numpy as np
 
 from ert_data import loader
 
+# importlib.metadata added in python 3.8
+try:
+    from importlib import metadata
+    __version__ = metadata.version("ert")
+except ImportError:
+    from pkg_resources import get_distribution
+    __version__ = get_distribution("ert").version
+
 
-class MeasuredData(object):
-    def __init__(self, facade, keys, index_lists=None, load_data=True):
-        self._facade = facade
-        self._set_data(self._get_data(keys, index_lists, load_data))
+class MeasuredData(pd.DataFrame):
+    def __init__(self, facade, keys, index_lists=None, load_data=True, case_name=None):
```

Comment:
Do we want to include a Doc string on the `MeasuredData` class as well? Feel free to do so as part of a rebase.